types/d3-dsv/index.d.ts

```diff
@@ -77,6 +77,7 @@ export interface DSVParsedArray<T> extends Array<T> {
  * The returned array also exposes a columns property containing the column names in input order (in contrast to Object.keys, whose iteration order is arbitrary).
  *
  * Equivalent to `dsvFormat(",").parse`.
+ * Note: requires unsafe-eval content security policy.
```

Comment:
Out of curiosity, why would formatting have security implications?

Reply:
> If a content security policy is in place, note that dsv.parse requires unsafe-eval in the script-src directive, due to the (safe) use of dynamic code generation for fast parsing.
([Source](https://github.com/d3/d3-dsv#content-security-policy))

Reply:
Wild